packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts

```diff
@@ -1,22 +1,60 @@
 import { fonts } from "@constants/font.constant";
 import styled from "@emotion/styled";
+import { media } from "@styles/media";
 import mixins from "@styles/mixins";
 
 export const AssetListHeaderWrapper = styled.div`
-  ${mixins.flexbox("row", "center", "flex-start")};
+  ${mixins.flexbox("row", "center", "space-between")};
   width: 100%;
-  margin-bottom: 24px;
+  ${media.mobile} {
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 8px;
+  }
+
+  .title-container {
+    ${mixins.flexbox("row", "center", "flex-start")};
+    width: 651px;
+    gap: 36px;
+    ${media.mobile} {
+      width: 328px;
+      justify-content: space-between;
+    }
+  }
+  .mobile-title-container {
+    ${mixins.flexbox("row", "center", "flex-start")};
+    gap: 12px;
+  }
+  .icon-wrap {
+    ${mixins.flexbox("row", "center", "center")};
+  }
+  .search-icon {
+    width: 24px;
+    height: 24px;
+    * {
+      fill: ${({ theme }) => theme.color.icon08};
+    }
+  }
+  .search-icon:hover {
+    * {
+      fill: ${({ theme }) => theme.color.icon02};
+    }
+  }
 
   h2 {
     ${fonts.h5};
     color: ${({ theme }) => theme.color.text02};
-    margin-right: 36px;
+    ${media.mobile} {
+      ${fonts.h6};
+    }
   }
+
   .right-section {
```

Comment:
Text alignment in Switch components is broken on tablet environments.

Reply:
reslove !